### ISSUE TITLE
When using the Chrome browser, NVDA speaks alert repeatedly when page loads on sites with Disqus comments

### ISSUE THREAD

**Issue by pbsinnett** [nvaccess/nvda#9338]
NVDA 2018.4.1, Chrome 72.0.3626.119, Windows 10 17763.316

When navigating to a site with Disqus comments, NVDA will repeatedly speak alert until all the Disqus portion of the page loads. This behavior doesn't allow the loaded part of the page to be read, due to the repeated interruptions of speech. One site where this behavior is exhibited is androidpolice.com. To reproduce this behavior, just click on any of the articles.

Would it be possible to filter out all the empty alerts, or ones from Disqus?

**Comment by SeanFarrow** [comment]
I can confirm this is also the case with NVDA 2018.4.1 and Chrome 72.0.3626.121 on Windows 10 version 1809 (OS Build 17763.348).

I've tested this on a completely different website (www.troyhunt.com) and what seems to be happening is an alert is spoken for each comment attached to a particular article.

I'm not sure this is an NVDA problem and it's probably something Disqus should be made aware of, although, in case this happens on other pages, it would be nice to be able to change how/when alerts are spoken.

I'm thinking both being able to suppress alerts on specific sites and also determine whether alerts would interupt reading in general. JAWS, for example has the concept of aria live regions speaking on a polite basis, perhaps something like this should be implemented?

**Comment by Adriani90** [comment]
@pbsinnett, @SeanFarrow I cannot reproduce with NVDA 2023.1 RC1 and Chrome 111. Are you still having this issue?

**Comment by pbsinnett** [comment]
No, it appears to be resolved for now.

**Comment by seanbudd** [comment]
closing as fixed
